apps/studio/prisma/types.ts

```diff
@@ -12,15 +12,22 @@ import type {
   IsomerSiteThemeProps as _IsomerSiteThemeProps,
   IsomerSiteWideComponentsProps as _IsomerSiteWideComponentsProps,
 } from "@opengovsg/isomer-components"
+import type { Tagged } from "type-fest"
 
 declare global {
   // eslint-disable-next-line @typescript-eslint/no-namespace
   namespace PrismaJson {
     // TODO: Rename all with XXXYYYJson instead of XXXJsonYYY
-    type SiteJsonConfig = _IsomerSiteConfigProps
-    type SiteThemeJson = _IsomerSiteThemeProps
-    type BlobJsonContent = _IsomerSchema
-    type NavbarJsonContent = _IsomerSiteWideComponentsProps["navBarItems"]
-    type FooterJsonContent = _IsomerSiteWideComponentsProps["footerItems"]
+    type SiteJsonConfig = Tagged<_IsomerSiteConfigProps, "JSONB">
```

Comment:
Can share why using tagged/unwraptagged is better here?

Reply:
Added an ADR 
https://www.notion.so/opengov/Explicitly-tagging-Kysely-jsonb-types-11577dbba78880c9917cd65dfe148e07?pvs=4